dataCollection/settings.py

```diff
@@ -8,7 +8,7 @@
 #TEMPLATE_DEBUG = DEBUG
 
 ADMINS = (
-    ('Tom Moore', 'tjm3772@rit.edu'),
+    ('Sally PandaFace', 'spf3000@rit.edu'),
```

Comment:
No, sorry. It is that I changed the file to contain fake data rather than real data.

The information put in the file will be relative to each server install. So the admin for the server will set the name and email to their own. On ours I have my name and email (should prolly put all 4 of us) as well as the actual password and username for our database.

Look at this file as a template :wink: